promgen/templates/promgen/service_detail.html

```diff
@@ -8,15 +8,96 @@
 
 {% block content %}
 
-<div class="page-header" v-pre>
-  <h1>Service: {{ service.name }}</h1>
-  {% if service.owner %}
-  <p>{% trans 'Contact' %}: {{service.owner.username}}</p>
-  {% endif %}
+<div class="page-header promgen-flex-space-between-center">
+  <div>
+    <h1>Service: {{ service.name }}</h1>
+    {% if service.owner %}
+    <p>{% translate 'Contact' %}: {{service.owner.username}}</p>
+    {% endif %}
+  </div>
+  
+  <div>
+
+    <div class="btn-group btn-group-sm" role="group" aria-label="...">
+      <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        {% translate "Actions" %} <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <li role="presentation"><a href="{% urlqs 'audit-list' service=service.id %}">{% translate "Edit History" %}</a></li>
+        <li role="presentation"><a href="{% urlqs 'alert-list' service=service.name %}">{% translate "Alert History" %}</a></li>
+
+        <hr>
+
+        <li role="presentation"><a href="{% url 'api:service-rules' name=service.name %}">{% translate "Export Rules" %}</a></li>
+        <li role="presentation"><a href="{% url 'api:service-targets' name=service.name %}">{% translate "Export Service" %}</a></li>
+
+        <hr>
+
+        <li role="presentation">
+          <a href="">
+          <form method="post" action="{% url 'service-delete' service.id %}" onsubmit="return confirm('{% translate "Delete this service?" %}')" style="display: inline">
+            {% csrf_token %}   
+              <button style="background:none;border: none;padding: 0;">{% translate "Delete Service" %}</button>  
+          </form>
+        </a>
+        </li>
+      </ul>
+    </div>
+
+    <a @click="setSilenceDataset" data-service="{{service.name}}" class="btn btn-warning btn-sm ml-2 mr-2">{% translate "Silence" %}</a>
+
+    <a href="{% url 'service-update' service.id %}" class="btn btn-primary btn-sm">{% translate "Edit Service" %}</a>
```

Comment:
```suggestion
    <a href="{% url 'service-update' service.id %}" class="btn btn-warning btn-sm">{% translate "Edit Service" %}</a>
```

In the rest of Promgen, we've used `btn-warning` for any editing action, so changing this to `btn-primary` seems strange . . . that said, having the orange silence button next to an orange edit button feels strange. We may want to re-consider our color conventions so that things are kept consistent across the app.